ApplicationLibCode/ProjectDataModel/RimContourMapProjection.cpp

```diff
@@ -391,7 +391,9 @@ void RimContourMapProjection::clearGridMapping()
     clearResults();
     clearTimeStepRange();
 
-    if ( m_contourMapProjection ) m_contourMapProjection->clearGridMapping();
+    if ( m_contourMapProjection ) m_contourMapProjection.reset();
```

Comment:
The if is redundant, ok to do reset() on a unique_pointer having nullptr as the contained object

Reply:
I agree. Fixed!